src/Component/QueryElevation.php

```diff
@@ -0,0 +1,345 @@
+<?php
+
+namespace Solarium\Component;
+
+use Solarium\Component\RequestBuilder\QueryElevation as RequestBuilder;
+
+/**
+ * QueryElevation component.
+ *
+ * @see https://lucene.apache.org/solr/guide/the-query-elevation-component.html
+ */
+class QueryElevation extends AbstractComponent
+{
+    /**
+     * Default options.
+     *
+     * @var array
+     */
+    protected $options = [
+        'handler' => 'elevate',
```

Comment:
I don't think that this is required. Or it's wrong!

According to the documentation the dedicated elevate handler exists in the techproducts example only "for brevity". But the component "may be added to any request handler".

Reply:
If you want to use Query Elevation on `/select`, you have to add it in `solrconfig.xml`. I (wrongfully?) assumed `/elevate` was always included and therefore it would "just work" this way even if you don't have access rights to edit the config.

I'll remove it and include a note about the config in the documentation. If you use the Solarium component on `/select` without adding the Solr component, all it does is cause head-scratching.